scripts/src/javatests/org/oppia/android/scripts/coverage/RunCoverageTest.kt

```diff
@@ -68,64 +105,120 @@ class RunCoverageTest {
     assertThat(exception).hasMessageThat().contains("No appropriate test file found")
   }
 
+  @Test
+  fun testRunCoverage_invalidFormat_throwsException() {
+    testBazelWorkspace.initEmptyWorkspace()
+    val exception = assertThrows<IllegalArgumentException>() {
+      main(tempFolder.root.absolutePath, "file.kt", "format=PDF")
+    }
+
+    assertThat(exception).hasMessageThat().contains("Unsupported report format")
+  }
+
+  @Test
+  fun testRunCoverage_ignoreCaseMarkdownArgument_returnsCoverageData() {
+    val filePath = "coverage/main/java/com/example/TwoSum.kt"
+
+    testBazelWorkspace.initEmptyWorkspace()
+    testBazelWorkspace.addSourceAndTestFileWithContent(
+      filename = "TwoSum",
+      testFilename = "TwoSumTest",
+      sourceContent = sourceContent,
+      testContent = testContent,
+      sourceSubpackage = "coverage/main/java/com/example",
+      testSubpackage = "coverage/test/java/com/example"
+    )
+
+    main(
+      "${tempFolder.root}",
+      filePath,
+      "format=Markdown",
+      "processTimeout=10"
+    )
+
+    val outputFilePath = "${tempFolder.root}" +
+      "$coverageDir/${filePath.removeSuffix(".kt")}/coverage.md"
+
+    assertThat(File(outputFilePath).exists()).isTrue()
+  }
+
+  @Test
+  fun testRunCoverage_ignoreCaseHTMLArgument_returnsCoverageData() {
```

Comment:
```suggestion
  fun testRunCoverage_ignoreCaseHtmlArgument_returnsCoverageData() {
```

Per https://developer.android.com/kotlin/style-guide#camel_case. Ditto elsewhere in this PR for identifiers.

Reply:
Thanks for the resource. Updated all occurrences as guided.